nautobot/extras/models/models.py

```diff
@@ -292,6 +292,17 @@ def clean(self):
         if self.file_extension.startswith("."):
             self.file_extension = self.file_extension[1:]
 
+        # Don't allow two ExportTemplates with the same name, content_type, and owner.
+        # This is necessary because Django doesn't consider NULL=NULL, and so if owner is NULL the unique_together
+        # condition will never be matched even if name and content_type are the same.
+        if ExportTemplate.objects.exclude(pk=self.pk).filter(
+            name=self.name,
+            content_type=self.content_type,
+            owner_content_type=self.owner_content_type,
+            owner_object_id=self.owner_object_id,
+        ):
```

Comment:
It's a little more efficient and readable to use `.exists()` for queries like this since the result set may grow over time and boolean checks evaluate the entire result set, while `exists` will limit it to a single object.

```suggestion
        ).exists():
```